backend/src/main/scala/cromwell/backend/callcaching/JobCachingActorHelper.scala

```diff
@@ -0,0 +1,36 @@
+package cromwell.backend.callcaching
+
+import akka.actor.Actor
+import cromwell.backend.{BackendConfigurationDescriptor, BackendJobDescriptor}
+import cromwell.core.logging.JobLogging
+
+/**
+  * Base trait for mixins related to caching functionality.
+  *
+  * This trait is extended by sub traits that provide more functionality for a specific backend. That sub trait  is
+  * then mixed into a pair of actors, the first that does the (async-)execution, the second that responds to cache hits.
+  *
+  * Very closely tied to, but not the same as a BackendLifecycleActor/BackendJobLifecycleActor. Those traits define and
```

Comment:
Not a typo. The `JobCachingActorHelper` is a mostly-dupe of the BJLA, without inheriting the BLA's additional functionality. That's where that errant "//For" comment came from. 😉 

```scala
trait BackendJobLifecycleActor extends BackendLifecycleActor with JobLogging {
  //For Logging and boilerplate
  override lazy val workflowId = jobDescriptor.workflowDescriptor.id
  override lazy val jobTag = jobDescriptor.key.tag

  protected def jobDescriptor: BackendJobDescriptor
}
```